src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceRentDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  let totalAmount = days * priceRentDay;
+
+  if (days >= longTerm) {
+    totalAmount -= longTermDiscount;
+  } else if (days >= shortTerm) {
+    totalAmount -= shortTermDiscount;
```

Comment:
Since you've removed the `else` keyword as suggested in the previous comment, you can now directly check for the `shortTerm` condition without the need for `else if`. Just use a separate `if` statement.